jsapp/js/account/security/apiToken/apiTokenSection.module.scss

```diff
@@ -0,0 +1,55 @@
+@use 'scss/_colors';
+@use 'scss/sizes';
+@use 'scss/libs/_mdl';
+
+.root {
+  margin-top: sizes.$x20;
+  margin-bottom: sizes.$x60;
+  padding-top: sizes.$x14;
+  display: flex;
+  border-top: sizes.$x1 solid;
+  border-color: colors.$kobo-gray-92;
+}
+
+.titleSection {
+  flex: 2;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.title {
+  margin: 0;
+  color: colors.$kobo-gray-40;
+  font-weight: 600;
+}
+
+.bodySection {
+  flex: 4;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+
+  label {
+    display: inline-block;
+    vertical-align: top;
+    width: 100%;
+
+    input {
+      font-family: mdl.$font_mono;
+    }
+  }
+}
+
+.optionsSection {
+  text-align: right;
+  // FIXME: Why does text-align do nothing?
+  display: ruby;
```

Comment:
This is the first time I see `ruby` 🧑‍🎓 So why is it here? I've read (right now) that this is being used mostly for annotations in east asian documents

Reply:
Perhaps it would make more sense if we'd use something more generic?

Reply:
But if not, try `ruby-align` instead of `text-align`

Reply:
Fixed this along with some other alignment issues in [`a39eeae46`](https://github.com/kobotoolbox/kpi/pull/4163/commits/a39eeae465a468ce1dfdb59b9e0b5919bc53745c)

After:
<img width="927" alt="After" src="https://user-images.githubusercontent.com/3514715/209415268-65942ee7-1809-4d09-971c-a7d45a6a945b.png">

<details><summary>Before:</summary>
<img width="928" alt="Before" src="https://user-images.githubusercontent.com/3514715/209415420-ee849c8b-88cb-449b-a4d3-7b5b6956ff01.png">

</details>